### PR TITLE
Add Accessibility docs

### PR DESCRIPTION
**Pull request by vine77**
### Description of proposed changes

Add "Accessibility Guide" docs, sourced from [the Google Doc](https://docs.google.com/document/d/1G02Q9muByvByX9QoWUxhmW9FUw4SWVHSdzNp6yRKeRo/edit?usp=sharing).

### Screenshot of proposed changes

Screen Shot 2022-06-23 at 12 36 58 PM

### CODE REVIEW

**Review by steveax — CHANGES_REQUESTED** [review]
Looks great, thanks! Left some comments around skipped heading levels.

https://www.w3.org/WAI/tutorials/page-structure/headings/

File: packages/design-system-website/foundations/Accessibility.md [hunk 1]
```diff
@@ -1 +1,101 @@
-[Add Accessibility content here]
+## Why is accessibility important?
```
[the remark this answers is not in the document]
Comment (vine77) [reply packages/design-system-website/foundations/Accessibility.md:1]: With Styleguidist, there ends up being an `h1` at the top generated from the config: https://github.com/puppetlabs/design-system/pull/581/files#diff-517a792e222ff0a45abde8f1d3e9db9e408eda423b5a3f615a51db843035fdb9R83

So in this PR at the moment it says "Accessibility Guide". ("Accessibility" might be simpler. 🙂)

File: packages/design-system-website/foundations/Accessibility.md [hunk 2]
```diff
@@ -1 +1,101 @@
...
+
+  Content must be robust enough that it can be interpreted reliably by a wide variety of user agents, including assistive technologies.
+
+Evaluate contributions through the lens of the four principles of accessibility. Consider users in a variety of situations with different needs and abilities. Can they interact with your component? Can they complete the workflow to achieve their goals?
+
+Read about the principles in more detail in the [WCAG documentation](https://www.w3.org/WAI/WCAG21/Understanding/intro#understanding-the-four-principles-of-accessibility).
+
+## Who benefits from accessible design?
+
+#### Accessible design helps everyone, including people with:
```
[the remark this answers is not in the document]
Comment (SaysKez) [reply packages/design-system-website/foundations/Accessibility.md]: Fixed! Thanks for pointing that out, Steve,

**Review by SaysKez — APPROVED** [review]
Updated the headings to avoid skipping a level and h1 will be generated from the config so looks like we're good to go. Thanks!

**Pull request merged**